Odata-docs/client/v7/navigation-properties.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Navigation Properties"
+description: "This tutorial describes how to use navigation properties on client side"
+
+author: mumbi-o
+ms.author: mowambug
+ms.date: 7/1/2019
```

Comment:
Is this a new page and are the date and author details correct?

Reply:
Yes, it's a new page.
Docs are owned by the PM. 
I will keep the date that way for now. I will update dates for all pages once we merge to the feature branch.